### PR TITLE
Fix various documentation errors

### PR DESCRIPTION
**Pull request by frangio**
Fixes LIB-1072

### CODE REVIEW

**Review by Amxx — APPROVED** [review]

File: contracts/governance/Governor.sol [hunk 1]
```diff
@@ -135,7 +135,6 @@ abstract contract Governor is Context, ERC165, EIP712, Nonces, IGovernor, IERC72
      * @dev See {IGovernor-state}.
      */
     function state(uint256 proposalId) public view virtual returns (ProposalState) {
-        // ProposalCore is just one slot. We can load it from storage to stack with a single sload
```
Comment (ernestognw) [inline contracts/governance/Governor.sol:138]: This is still true. Are we just considering it not worth to keep the comment?

File: contracts/utils/structs/DoubleEndedQueue.sol [hunk 2]
```diff
@@ -50,6 +47,8 @@ library DoubleEndedQueue {
 
     /**
      * @dev Inserts an item at the end of the queue.
+     *
+     * Reverts with `QueueFull` if the queue is full.
```
Comment (ernestognw) [inline contracts/utils/structs/DoubleEndedQueue.sol:51]:
```suggestion
     * Reverts with {QueueFull} if the queue is full.
```

File: contracts/utils/structs/DoubleEndedQueue.sol [hunk 3]
```diff
@@ -78,6 +77,8 @@ library DoubleEndedQueue {
 
     /**
      * @dev Inserts an item at the beginning of the queue.
+     *
+     * Reverts with `QueueFull` if the queue is full.
```
Comment (ernestognw) [inline contracts/utils/structs/DoubleEndedQueue.sol:81]:
```suggestion
     * Reverts with {QueueFull} if the queue is full.
```

**Review by ernestognw — COMMENTED** [review]
Minor nits, LGTM

**Review by ernestognw — APPROVED** [review]

File: contracts/governance/Governor.sol (hunk 1 above)
Comment (frangio) [reply contracts/governance/Governor.sol:138]: ProposalCore is two slots now, but the rest of the comment is sort of true and relevant so I'll add it back.

**Pull request merged**